.github/workflows/ruff.yml

```
@@ -0,0 +1,15 @@
+name: Ruff
+on: [ push, pull_request ]
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)